setup.cfg

```diff
@@ -11,7 +11,6 @@ classifier =
     Programming Language :: Python :: 2.7'
     Programming Language :: Python :: 3.5'
     Programming Language :: Python :: 3.6'
-    Programming Language :: Python :: 3.7'
```

Comment:
just making sure that removing 3.7 is intentional.

Reply:
Yep, 3.7 fails to build with Travis right now.